app/util/data_preparation/jira_prepare_data.py

```diff
@@ -153,6 +153,14 @@ def __check_current_language(jira_api):
                          f'Please change your profile language to "English (United States) [Default]"')
 
 
+def __check_for_admin_permissions(jira_api):
+    user_permissions = jira_api.get_user_permissions()
+    if not any((user_permissions['permissions']['ADMINISTER']['havePermission'],
```

Comment:
It would be more crear to use `or` instead of any()
```
    if not (user_permissions['permissions']['ADMINISTER']['havePermission'] or
            user_permissions['permissions']['SYSTEM_ADMIN']['havePermission']):
```

Reply:
no problem, let's make it more readable